aptos-move/aptos-aggregator/src/resolver.rs

```diff
@@ -18,29 +28,89 @@ pub enum AggregatorReadMode {
 /// Returns a value of an aggregator from cache or global storage.
 ///   - Ok(..)       if aggregator value exists
 ///   - Err(..)      otherwise.
-pub trait AggregatorResolver {
+pub trait TAggregatorView {
+    type Identifier;
+
+    fn get_aggregator_v1_state_value(
+        &self,
+        id: &Self::Identifier,
+    ) -> anyhow::Result<Option<StateValue>>;
+
+    fn get_aggregator_v1_value(&self, id: &Self::Identifier) -> anyhow::Result<u128> {
+        let maybe_state_value = self.get_aggregator_v1_state_value(id)?;
+        // TODO: consider reviving Option<u128>?
+        let u128_bytes = maybe_state_value
+            .expect("Aggregator V1 cannot be deleted")
```

Comment:
can it not be deleted? I keep mixing this up?

Reply:
Had a question above, I think it still can? cc @igor-aptos @vusirikala ? We still have delete in the change set as well...

Reply:
Returned deletion for now